docs/user_guide/index.rst

```diff
@@ -8811,6 +8811,221 @@ Environment Code to a target node with Action code, the issue will be
+first. 
+
+To discard the assignment of a business term from a BIE:
+
+1. On the top menu of the page, click "BIE".
+   
+2. Choose "View/Edit BIE" from the drop-down list. 
+   
+3. Locate the BIE you want to see the business term assignments. Use the *DEN*, *Business Context*, *Branch*, *State*,
+   *Owner*, *Updater*, *Updated start date*, or *Updated end date* search filters to help locate the desired BIE. 
+   (see `How to use Search Filters <#how-to-use-search-filters>`__). Click on the BIE DEN to open its "Edit BIE" page. 
+
+4. Expand the root node on the BCC (for BBIE) or ASCC (for ASBIE) tree in the left navigation panel.  
+
+5. Select a desired child property node (an ASCCP node in bolded blue font or a BCCP node in regular green font). 
+   The detail for that child node is displayed in the right panel. The *Used* checkbox must be checked. 
+
+6. Click "Show Business Terms" button in the right panel.
+   
+7. On the newly-opened "Business Term Assignement" page, the selected BIE is displayed right before the "Turn off" button. 
+   All the business terms assigned for the selected BIE are displayed in the table below the "Search" button. 
+
+8. Located the business term to be discarded from the assignment. Use the *Business Term*, *External Reference URI*, *Type Code*
+   , *Updater*, *Updated start date*, or *Updated end date* search filters to help locate the desired business term. 
+   Note that *Preferred Only* checkbox can help filter efficiency too. 
+
+9. Click "Search" button. 
+    
+10. Select the desired business term. The "Discard" button at the top right of the page will be enabled. Click "Discard",  and a 
+    dialog is open where you can confirm or cancel the request. Only the assignment for this given BIE is permanently removed. 
+    The business term is still displayed in "View/Edit Business Term" page. 
+
+Load Business Terms from external source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the preferred way for the end users to create business terms in Score. This method allows the end users to bulk upload 
+business terms from an external csv file. 
+
+To upload from an external file:
+
+1. On the top menu of the page, click "BIE".
+   
+2. Choose "View/Edit Business Term" from the drop-down list. 
+
+3. On the returned "Business Term" page, click "Upload Business Terms" button at the top right corner of the page. 
+   
+4. On the returned "Upload Business Terms" page, click "Download template" button at the top right corner of the page. 
+   A csv template file named "businessTermTemplateWithExample" will be saved into your "Download" folder on your local 
+   computer. 
+
+5. Fill out all your business terms in the downloaded csv file. Make sure that the format for each column is correct for each 
+   business term. Note that *businessTerm* and *externalReferenceUri* columns are required. Save all your changes. 
```

Comment:
"Make sure that the format for each column is correct for each business term" => what is the correct format? You need to tell the user.